client/src/mvc/rules/rule-definitions.js

```diff
@@ -270,13 +270,19 @@ const RULES = {
                 rule.replacement = component.addColumnRegexReplacement;
             }
             if (component.addColumnRegexGroupCount) {
-                rule.group_count = component.addColumnRegexGroupCount;
+                rule.group_count = parseInt(component.addColumnRegexGroupCount);
```

Comment:
Do you know why is this not an integer in the first place ?

Reply:
Well that is a good question. Digging a bit deeper, probably this https://stackoverflow.com/a/49749196. I wonder if that changes with some Vue upgrade at some point. I swear the screenshots would indicate this worked at some point.

Reply:
I've got a commit for dev anyway - I'll see if I can include a template fix for this as part of that.